### PR TITLE
Get spellcheck workflow

### PR DESCRIPTION
**Pull request by cjeanner**
As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running

### CODE REVIEW

**Comment by cjeanner** [comment]
@Sandeepyadav93 @lewisdenny any thoughts on that PR? We shouldn't get THAT many doc update, meaning new content SHOULD be small enough to not be a burden to spell-check... ?

File: .github/workflows/pyspell.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# This is workflow for spell checking using PySpelling lib (https://pypi.org/project/pyspelling/)
+name: Spellcheck
+# Controls when the action will run.
+on:
```
Comment (Sandeepyadav93) [inline .github/workflows/pyspell.yml:4]: @cjeanner I wonder if we want to optimize this job to only run when we change README.md using the path filter.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

but this is a light job we can run on all changes too.

Comment (cjeanner) [reply .github/workflows/pyspell.yml:4]: it's now filtered on only .md files. Thanks for the headsup!

**Comment by lewisdenny** [comment]
> @Sandeepyadav93 @lewisdenny any thoughts on that PR? We shouldn't get THAT many doc update, meaning new content SHOULD be small enough to not be a burden to spell-check... ?

I think a very small burden is worth professional looking docs. Now the typos are fixed, going forward it will be no issue :)

**Comment by cjeanner** [comment]
recheck

**Comment by Sandeepyadav93** [comment]
/lgtm

thanks cedric

**Comment by cjeanner** [comment]
recheck

**Review by raukadah — APPROVED** [review]
/approve

**Pull request merged**